.bacon.yml

```diff
@@ -62,3 +62,32 @@ test_suites:
       script_name: unit-karma
       criteria: MERGE
       queue_name: small
+
+    - name: v3-lint
+      script_path: /root/okta/okta-signin-widget/src/v3/scripts
+      sort_order: '10'
+      timeout: '10'
+      script_name: lint
+      criteria: OPTIONAL
+      queue_name: small
+    - name: v3-unit-jest
+      script_path: /root/okta/okta-signin-widget/src/v3/scripts
+      sort_order: '11'
+      timeout: '15'
+      script_name: unit-jest
+      criteria: OPTIONAL
+      queue_name: small
+    - name: v3-e2e
+      script_path: /root/okta/okta-signin-widget/src/v3/scripts
+      sort_order: '12'
+      timeout: '15'
+      script_name: e2e
+      criteria: OPTIONAL
+      queue_name: small
+    - name: v3-publish
+      script_path: /root/okta/okta-signin-widget/src/v3/scripts
+      sort_order: '13'
+      timeout: '60'
+      script_name: publish
+      criteria: OPTIONAL
```

Comment:
Should these stay "optional"?

Reply:
They're optional right now so as not to impede v2 development

Reply:
I'm OK with this (it makes sense). Is there any way to have Bacon selectively enforce certain test suites only if certain directories are touched? Might be a question for productivity team. Ideally we don't have to self-enforce on our PRs.